src/popup/extract_path.js

```diff
@@ -2,20 +2,31 @@ var Popup = Popup || {};
 
 // Extract the relevant path from a location, such as `/foo` from URLs like
 // `www.gov.uk/foo` and `www.gov.uk/api/content/foo`.
-Popup.extractPath = function(location) {
+Popup.extractPath = function(location, renderingApplication) {
```

Comment:
@tijmenb The user isn't being redirected. The renderingApplication is just being used to populate links to the content api and the external links to github for the smart-answer regardless of where you are in the smart-answer flow.

The problem we had was getting the links to the content api to work. If you're in the middle of a smartanswer, you can't just add `/api/content` to the middle of the path, a 404 will be thrown. e.g https://www.gov.uk/api/content/marriage-abroad/y/france/marriage.

Initially we did try doing all of the work in the generateContentToolsLinks function, but this required doing another call to the content store. See commit [e40508c](https://github.com/ikennaokpala/govuk-toolkit-chrome/blob/e40508ce5647e754eebf0157536926f591bcd5f1/src/popup/content_tools_links.js).

In a separate, but related note, we're don't currently allow people to view the govspeak versions of smart-answers in the production and staging environments.  That decision will be revisited if the Content designers are happy with the changes.